flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/source/proxy/DynamoDbStreamsProxy.java

```diff
@@ -127,8 +153,8 @@ private DescribeStreamResponse describeStream(String streamArn, @Nullable String
                 dynamoDbStreamsClient.describeStream(describeStreamRequest);
 
         StreamStatus streamStatus = describeStreamResponse.streamDescription().streamStatus();
-        if (!(streamStatus.equals(StreamStatus.DISABLED)
-                || streamStatus.equals(StreamStatus.ENABLING.toString()))) {
+        if (streamStatus.equals(StreamStatus.DISABLED)
```

Comment:
As discussed offline - this should only be `ENABLING`

Reply:
fixed